android/libraries/rib-base/src/main/java/com/badoo/ribs/android/recyclerview/RecyclerViewHostFeature.kt

```diff
@@ -0,0 +1,84 @@
+package com.badoo.ribs.android.recyclerview
+
+import android.os.Parcelable
+import com.badoo.mvicore.element.Bootstrapper
+import com.badoo.mvicore.element.Reducer
+import com.badoo.mvicore.element.TimeCapsule
+import com.badoo.mvicore.feature.ReducerFeature
+import com.badoo.ribs.android.recyclerview.RecyclerViewHost.Input
+import com.badoo.ribs.android.recyclerview.RecyclerViewHostFeature.State
+import com.badoo.ribs.core.routing.configuration.ConfigurationKey
+import io.reactivex.Observable
+import kotlinx.android.parcel.Parcelize
+import java.util.UUID
+
+private val timeCapsuleKey = RecyclerViewHostFeature::class.java.name
```

Comment:
Use a constant string to avoid having a random string clash with obfuscation?

Reply:
Good catch, thanks.